tests/foreman/cli/test_provisioning.py

```diff
@@ -15,15 +15,423 @@
+    dom.location = dom.location + [loc]
+    dom = dom.update(['organization', 'location'])
+    return dom
+
+
+@pytest.fixture(scope='session')
+def capsule(user_credentials, org, loc):
+    capsule = entities.SmartProxy(id=1).read()
+    if org not in capsule.organization:
+        capsule.organization.append(org)
+    if loc not in capsule.location:
+        capsule.location.append(loc)
+    capsule = capsule.update(['organization', 'location'])
+    return capsule
+
+
+@pytest.fixture(scope='session')
+def subnet(user_credentials, org, loc, domain, capsule):
+    subnet_name = gen_string('alpha')
+    subnet = entities.Subnet(
+        user_credentials,
+        name=subnet_name,
+        location=[loc],
+        organization=[org],
+        ipam=settings.vlan_networking.dhcp_ipam,
+        network=settings.vlan_networking.subnet,
+        gateway=settings.vlan_networking.gateway,
+        from_=settings.vlan_networking.dhcp_from,
+        to=settings.vlan_networking.dhcp_to,
+        mask=settings.vlan_networking.netmask,
+        dns_primary=settings.vlan_networking.dns_primary,
+        domain=[domain],
+        dns=capsule,
+        dhcp=capsule,
+        discovery=capsule,
+        # httpboot=capsule,   # TODO
+        template=capsule,
+        tftp=capsule,
+        remote_execution_proxy=[capsule],
+    ).create()
+    return subnet
+
+
+@pytest.fixture(scope='session')
+def lce(user_credentials, org):
+    return entities.LifecycleEnvironment(
+                user_credentials,
+                organization=org).create()
+
+
+@pytest.fixture(scope='session')
+def rhel7(user_credentials, org):
+    repo_id = enable_rhrepo_and_fetchid(
+        basearch='x86_64',
+        org_id=org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhel7']['name'],
+        reposet=REPOSET['rhel7'],
+        releasever=REPOS['rhel7']['releasever'],
+    )
+    repo = entities.Repository(user_credentials, id=repo_id)
+    response = repo.sync(synchronous=False)
+    entities.ForemanTask(id=response['id']).poll(timeout=1800)
+    return repo.read()
+
+
+@pytest.fixture(scope='session')
+def rhel7ks(user_credentials, org):
+    repo_id = enable_rhrepo_and_fetchid(
+        basearch='x86_64',
+        org_id=org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhel7ks']['name'],
+        reposet=REPOSET['rhel7ks'],
+        releasever=REPOS['rhel7ks']['releasever'],
+    )
+    repo = entities.Repository(user_credentials, id=repo_id)
+    response = repo.sync(synchronous=False)
+    entities.ForemanTask(id=response['id']).poll(timeout=1800)
+    return repo.read()
+
+
+@pytest.fixture(scope='session')
+def rhel8ks(user_credentials, org):
+    # First RHEL8 AppStream KS
+    repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=org.id,
+            product='Red Hat Enterprise Linux for x86_64',
+            repo='Red Hat Enterprise Linux 8 for x86_64 - AppStream Kickstart x86_64 8.0',
```

Comment:
@jhutar It would be great if this goes to constants.py. Same for baseos repo.

Reply:
Thank you - solved in:
```
[rhel8_provisioning 4182017e] Move constants into constatns.py
```